internal/node/controller/routes_test.go

```diff
@@ -228,12 +228,6 @@ func TestGetNodeHandler(t *testing.T) {
 
 	var respBody types.GetNodeResponse
 	require.NoError(t, json.Unmarshal(bytes, &respBody))
-	for k, v := range respBody.State {
```

Comment:
why did this get deleted?

Reply:
I don't think this for loop was doing anything real, it's just looping through each key and value, and then accessing those keys in the map and checking the value matches.